src/features/userSlice.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Importing the store from a module can cause cyclic dependency issues. Consider restructuring your app to avoid this.